pandasaurus_cxg/anndata_enricher.py

```diff
@@ -15,15 +17,19 @@ def __init__(
         file_path: str,
         cell_type_field: Optional[str] = "cell_type_ontology_term_id",
         context_field: Optional[str] = "tissue_ontology_term_id",
+        ontology_list_for_slims: Optional[List[str]] = ["Cell Ontology"],
```

Comment:
We should validate if all ontology titles are valid and raise an exception otherwise.

Reply:
Will having a Enum for ontology titles help users to find the correct ontology names?

```python
from enum import Enum

class OntologyNames(Enum):
    CL = "Cell Ontology"
    ENVO = "The Environment Ontology"
    DPO= "Drosophila Phenotype Ontology (DPO)"
    ....
```

Then users can provide the list as `[OntologyNames.CL.value, OntologyNames.ENVO.value]`. If ubergraph is extended, users can still use the string values.

Reply:
In this particular case, I think we only care about CL but want to leave an option to over-ride.

Validation should be in pandasaurus itself.  Pandasaurus also needs more support for letting users know what is possible.  We could have a method that lists all ontologies (referencing that in docstring of method for listing slims.)

It looks like your method takes dc:title of ontology. Easy to retrieve a list of these (although be careful about consistency dc namespace).

Reply:
> It looks like your method takes dc:title of ontology. Easy to retrieve a list of these (although be careful about consistency dc namespace).

This is the query I'm using to retrieve slim details; https://api.triplydb.com/s/lFTW8trIj.

Reply:
I have created https://github.com/INCATools/PandaSaurus/issues/21 to keep track